literature.bib

```diff
@@ -3515,7 +3592,7 @@ @inproceedings{BL:FOSD14
 	keywords = {model-based software engineering, software product lines, test case generation},
 	tt-tags = {SPL Testing},
 	tc-tags = {classified by Thomas,testing,sample-based analysis,family-based specification,implementation independent,design,program,no tool,no evaluation,no proof,subsumed by none},
-	sampling-tags = {name=incremental test-suite optimization,test artifacts,greedy,requirements coverage,no tool,no evaluation,no proof,testing},
+	sampling-tags = {name=incremental test-suite optimization,feature model,greedy,requirements coverage,unavailable tool,no evaluation,no proof,testing},
```

Comment:
why that change? any evidence?

Reply:
Cited from the paper: "In the following,  we propose an incremental approach for test suite optimization which solely requires the FM and a 150% test model as inputs." 
Therefore, I added the feature model as input. Also, double-checked and test artifacts are still required. Thus, I added the tag test artifacts again.

I also changed no tool to unavailable to for consistency with other classified papers